src/js/elements/mapchip/mapchip.js

```diff
@@ -91,6 +190,33 @@ export class MapChip extends Component {
         this.title = label;
     }
 
+    setFilterLabelTotalCount(groups) {
+        const total_group_length = groups.length - 1;
+        this._totalFilterCountContainer.html(total_group_length);
+    }
+
+    setFilterLabelSelectedCount(selectedFilter) {
+        const selected_filter_length = Object.keys(selectedFilter).length;
+        this._selectedFilterCountContainer.html(selected_filter_length);
+    }
+
+    setFilterLabelContainerVisibility(is_visible=!this.isContentVisible) {
```

Comment:
can you please change the variable name as `isVisible` instead of `is_visible`. we generally use camelCase instead of snake_case

Reply:
Sorry habit of writing 🐍 😅 
Will fix this